tslint.json

```diff
@@ -30,6 +30,7 @@
     "trailing-comma": [
       true,
       { "multiline": "always", "singleline": "never" }
-    ]
+    ],
+    "no-unused-variable": [true, {"ignore-pattern": "React" }]
```

Comment:
React + JSX, because JSX magically adds the `React.createComponent` - you still include it but it may not be referred to in code